pilot/pkg/model/push_context_test.go

```diff
@@ -461,6 +461,18 @@ func TestIsClusterLocal(t *testing.T) {
 			host:     "s.kube-system.svc.cluster.local",
 			expected: true,
 		},
+		{
+			name:     "discovery server is local by default",
+			m:        mesh.DefaultMeshConfig(),
+			host:     "istiod.istio-system.svc",
+			expected: true,
+		},
+		{
+			name:     "discovery server is local with domain suffix",
```

Comment:
@nmittler when checking "IsClusterLocal" would we expect that they will always include the domain suffix?

Reply:
I believe so, are you not seeing that?

Reply:
I'm only seeing that, I was just seeing if we need to be robust to both with and without.

Reply:
No, I think everything should be fqdn, so we're good.